tar/siginfo.c

```diff
@@ -51,6 +51,15 @@ struct siginfo_data {
 	/* How large is the archive entry? */
 	int64_t size;
 
+	/* How many files in total? */
```

Comment:
s/files in total/files have we handled in total/ ?

Or is there a reason this comment is different from the other two?